opensearch_dashboards.json

```diff
@@ -12,5 +12,7 @@
     "dashboard",
     "opensearchUiShared"
   ],
-  "optionalPlugins": ["dataSource", "dataSourceManagement"]
+  "optionalPlugins": ["dataSource", "dataSourceManagement"],
+  "supportedOSDataSourceVersions": ">=2.9.0",
+  "requiredOSDataSourcePlugins": ["opensearch-ml"]
```

Comment:
Are these key names defined by each plugins? Is it standardized?

Reply:
Yeah. These key names were been introduced in these two PRs:
https://github.com/opensearch-project/OpenSearch-Dashboards/pull/7146
https://github.com/opensearch-project/OpenSearch-Dashboards/pull/7100
They are optional properties, all plugins should using the same key names.